cve-resolution-strategy.gradle

```diff
@@ -0,0 +1,89 @@
+configurations.all {
+    resolutionStrategy {
+        eachDependency { DependencyResolveDetails det ->
+            if (det.requested.name == 'tomcat-embed-core') {
```

Comment:
As discussed please do the clustering of conditions and add appropriate comments.

Reply:
fixes are grouped and added the comments